src/app.controller.ts

```diff
@@ -12,4 +13,10 @@ export class AppController {
   getHello(): string {
     return this.appService.getHello();
   }
+
+  @Get("protected")
```

Comment:
Small stylistic issue, but mixing of quotes. Here it's double, above it's single. We should probably agree on which to be consistent throughout the project.